components/file_attachment/file_attachment.test.tsx

```diff
@@ -16,118 +16,130 @@ jest.mock('utils/utils.jsx', () => {
-            id: 1,
+            ...baseFileInfo,
             extension: 'svg',
             name: 'test.svg',
             width: 600,
             height: 400,
             size: 100,
         };
-        const wrapper = shallow(createComponent({fileInfo}));
+        const props = {...baseProps, fileInfo};
+        const wrapper = shallow(<FileAttachment {...props}/>);
         expect(wrapper).toMatchSnapshot();
     });
 
     test('should match snapshot, after change from file to image', () => {
         const fileInfo = {
-            id: 2,
+            ...baseFileInfo,
             extension: 'png',
             name: 'test.png',
             width: 600,
             height: 400,
             size: 100,
         };
-        const wrapper = shallow(createComponent());
-        wrapper.setProps({fileInfo});
+        const wrapper = shallow(<FileAttachment {...baseProps}/>);
+        wrapper.setProps({...baseProps, fileInfo});
         expect(wrapper).toMatchSnapshot();
     });
 
     test('should match snapshot, with compact display', () => {
-        const wrapper = shallow(createComponent({compactDisplay: true}));
+        const props = {...baseProps, compactDisplay: true};
+        const wrapper = shallow(<FileAttachment {...props}/>);
         expect(wrapper).toMatchSnapshot();
     });
 
     test('should match snapshot, without compact display and without can download', () => {
-        const wrapper = shallow(createComponent({canDownloadFiles: false}));
+        const props = {...baseProps, canDownloadFiles: false};
+        const wrapper = shallow(<FileAttachment {...props}/>);
         expect(wrapper).toMatchSnapshot();
     });
 
     test('should match snapshot, file with long name', () => {
         const fileInfo = {
-            id: 1,
+            ...baseFileInfo,
             extension: 'pdf',
             name: 'a-quite-long-filename-to-test-the-filename-shortener.pdf',
-            size: 100,
+
         };
-        const wrapper = shallow(createComponent({fileInfo}));
+        const props = {...baseProps, fileInfo};
+        const wrapper = shallow(<FileAttachment {...props}/>);
         expect(wrapper).toMatchSnapshot();
     });
 
     test('should match snapshot, when file is not loaded', () => {
-        const wrapper = shallow(createComponent());
+        const wrapper = shallow(<FileAttachment {...baseProps}/>);
         wrapper.setState({loaded: false});
         expect(wrapper).toMatchSnapshot();
     });
 
     test('should blur file attachment link after click', () => {
-        const wrapper = mountWithIntl(createComponent({compactDisplay: true}));
+        const props = {...baseProps, compactDisplay: true};
+        const wrapper = mountWithIntl(<FileAttachment {...props}/>);
+        const mockedTarget = document.createElement('div');
+        mockedTarget.blur = jest.fn();
         const e = {
             preventDefault: jest.fn(),
-            target: {blur: jest.fn()},
+            target: mockedTarget
```

Comment:
I didn't realize this was being passed in here from the test. I think it's better to avoid using `document` in the test code if possible, so the `if 'blur' in target` code is fine in the source. Sorry for the back-and-forth here.

Can you please also add the trailing comma here?